docs/content/manual/release-specific/v1.2.3/test-scalability-with-backing-image.md

```diff
@@ -0,0 +1,338 @@
+              mountPath: /usr/share/nginx/html9
+            - name: www10
+              mountPath: /usr/share/nginx/html10
+      volumeClaimTemplates:
+      - metadata:
+          name: www1
+        spec:
+          accessModes: [ "ReadWriteOnce" ]
+          storageClassName: "longhorn-test1"
+          resources:
+            requests:
+              storage: 32Mi
+      - metadata:
+          name: www2
+        spec:
+          accessModes: [ "ReadWriteOnce" ]
+          storageClassName: "longhorn-test1"
+          resources:
+            requests:
+              storage: 32Mi
+      - metadata:
+          name: www3
+        spec:
+          accessModes: [ "ReadWriteOnce" ]
+          storageClassName: "longhorn-test1"
+          resources:
+            requests:
+              storage: 32Mi
+      - metadata:
+          name: www4
+        spec:
+          accessModes: [ "ReadWriteOnce" ]
+          storageClassName: "longhorn-test1"
+          resources:
+            requests:
+              storage: 32Mi
+      - metadata:
+          name: www5
+        spec:
+          accessModes: [ "ReadWriteOnce" ]
+          storageClassName: "longhorn-test1"
+          resources:
+            requests:
+              storage: 32Mi
+      - metadata:
+          name: www6
+        spec:
+          accessModes: [ "ReadWriteOnce" ]
+          storageClassName: "longhorn-test1"
+          resources:
+            requests:
+              storage: 32Mi
+      - metadata:
+          name: www7
+        spec:
+          accessModes: [ "ReadWriteOnce" ]
+          storageClassName: "longhorn-test2"
+          resources:
+            requests:
+              storage: 32Mi
+      - metadata:
+          name: www8
+        spec:
+          accessModes: [ "ReadWriteOnce" ]
+          storageClassName: "longhorn-test3"
+          resources:
+            requests:
+              storage: 32Mi
+      - metadata:
+          name: www9
+        spec:
+          accessModes: [ "ReadWriteOnce" ]
+          storageClassName: "longhorn-test4"
+          resources:
+            requests:
+              storage: 32Mi
+      - metadata:
+          name: www10
+        spec:
+          accessModes: [ "ReadWriteOnce" ]
+          storageClassName: "longhorn-test5"
+          resources:
+            requests:
+              storage: 32Mi
+    ```
+    
+   Notice:  
+     - Each replica/pod of this StatefulSet contains 10 volumes. You just adjust the volume count by scaling this StatefulSet.
+     - Launching this StatefulSet will take a long time.
+5. Wait for all pods up and running.
+6. Create and attach a volume via UI. This should not take a long time.
+7. Upgrade the longhorn manager version to `v1.2.3-rc1`. Notice that each longhorn manager pod will take some time to walk through the ugprade path. Please do not operate Longhorn before each manager pod printing out the log like `"Debug Server listening on 127.0.0.1:6060" node=shuo-k8s-worker1`.
```

Comment:
For step7 to 8, I think we don't need to include in this test case, but just have a note this test case will fail in the previous version (< 1.2.3 or 1.3.0 if the issue does not be fixed in 1.2.3)

Reply:
Got it. I will remove these 2 steps then add more comments for explanation.